app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module AssessmentTasks
+  extend ActiveSupport::Concern
+
+  class AssessAgainstLegislationPresenter < PlanningApplicationPresenter
```

Comment:
Does this need to inherit from `PlanningApplicationPresenter`?

Reply:
Yeah I think so to call `super(template, planning_application)`